frontend/src/Components/EventRender.jsx

```diff
@@ -27,6 +27,7 @@ export default function EventRender(props) {
             .then(response => {
                 if (isMounted) {
                     setSingleEvent(response.data.payload);
+                    console.log(response.data.payload);
```

Comment:
intentional console.log or not?

Reply:
i'm just quickly scanning this pr for now because there are a number of conflicts